resources/wikia/modules/krux.js

```diff
@@ -0,0 +1,64 @@
+/*global Krux,define*/
+window.Krux || ((Krux = function () {
+	Krux.q.push(arguments);
+}).q = []);
+
+define('wikia.krux', ['wikia.window'], function (win) {
+	'use strict';
+
+	var maxNumberOfKruxSegments = 27;
+
+	function load(confid) {
```

Comment:
I think we did want to load the Krux script on every article view? 
@hakubo (otherwise Krux will only track the first PV)

On 03/12/2015 01:12 PM, Andrzej wrote:
> In resources/wikia/modules/krux.js
> <https://github.com/Wikia/app/pull/6605#discussion_r26296428>:
>
>> @@ -0,0 +1,64 @@
>> +/*global Krux,define*/
>> +window.Krux || ((Krux = function () {
>> +	Krux.q.push(arguments);
>> +}).q = []);
>> +
>> +define('wikia.krux', ['wikia.window'], function (win) {
>> +	'use strict';
>> +
>> +	var maxNumberOfKruxSegments = 27;
>> +
>> +	function load(confid) {
>
> @rogatty <https://github.com/rogatty> has a good point here! We don't
> want to put Krux script every time we call |load()| and we do it every
> time we change article in Mercury. My idea is to split this method into
> two: |loadScript()| and |exportPageParams()|. Mercury will call
> |krux.loadScript()| only with the first page load and then after
> changing article it'll just call |krux.exportPageParams()|. What are
> your thoughts @dianafa <https://github.com/dianafa> and @gabrys
> <https://github.com/gabrys>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/6605/files#r26296428>.
>